### ISSUE TITLE
Asking for more capacity to manipulate the location of where to create a Day Planner inside a daily note, eg inside a call-out

### ISSUE THREAD

**Issue by Duifkruid** [ivan-lednev/obsidian-day-planner#266]
Thank you for taking over this plugin, it is much used here and I really like your enhancements. One thing I do miss however, is for the plugin to find the the day's Day Planner inside a call-out or admonition. I rely on these heavily for reasons of privacy at work, and for closing off-focus items. I tried to tamper with the planner heading in settings, but as soon as it is something like ` ```ad-day`, the plugin can't find it. 

Screenshot of tragically missing Day Planner in right-hand pane time-line below. Excuse the double title: the admonition already fulfills the title Day Planner, but I hoped to lure the plugin to find the Day Planner by calling it a readable `## Day Planner`. 

![day-planner-admonition](https://github.com/ivan-lednev/obsidian-day-planner/assets/59362789/846ebb92-5dae-4c9b-92df-0c437371d32f)

Manually adding a new event inside the time-line, results in an extra heading of `## ```ad-day` or `## ``ad-day` at the bottom of the note, as I set the lever of zoom to two, and the heading to either ` ``ad-day`, or ` ```ad-day`. So it does this, in stead of finding the existing list. I'd really like it to be able to find the list itself, or rather the admonition. It would also be very helpful to format the place a newly created heading goes, in stead of defaulting to the bottom of the note; similar to e.g. QuickAdd, where it is possible to format in great detail what shape headings are, underneath which things are to happen, or where to put them if they do not exist yet. 

![day-planner-settings](https://github.com/ivan-lednev/obsidian-day-planner/assets/59362789/b372c333-c547-4e17-8d41-4ed6293d3f92)

Now where it showed up by letting it find "``ad-day", it does not when I use the appropriate three ticks to turn it into an admonition. Note the two ticks in stead of three in the screenshot of the settings; this is the work-around, I add a template to await content and then change into admonition. But of course, then the Day Planner gets erased from the time-line in the right-hand pane. 

All above apply equally to the formatting of call-outs: `>[!note]`. 

Tl;dr: Bring back the capacity to make use of admonitions or call-outs to stash the Day Planner, inside a daily note. It would also be very helpful to be able to choose under which heading a new Day Planner is created, when one does not exist yet. It currently is possible to choose the heading, but as soon as something changes, the plugin creates a new Day Planner at the bottom of the note.

Thank you for considering, and again for taking care of the plugin :)

**Comment by ivan-lednev** [comment]
Hello, @Duifkruid, please update and re-check. Make sure the Dataview plugin is installed.

**Comment by Duifkruid** [comment]
Thanks for looking into it Ivan. It didn't work and I felt stupid, added it onto a todo-list of action points but never got around to it. I see another encounters the same problem, I am hoping you will revisit the problem. Dataview is very much installed, I could not live without it.
